### PR TITLE
Dvc exp show deps

### LINKED ISSUE #3211: Document the color and order of columns in `exp show` table
> We don't even document the color or order of columns in exp show today AFAIK.

👍 Will try to cover as part of the docs P.R. for https://github.com/iterative/dvc/pull/6434

_Originally posted by @daavoo in https://github.com/iterative/dvc/issues/7304#issuecomment-1021064429_

### PR DESCRIPTION
**Pull request by daavoo**
P.R. for https://github.com/iterative/dvc/pull/7089

Closes #3211

### CODE REVIEW

**Comment by jorgeorpinel** [comment]
Looks like a change with a big impact on existing docs... @daavoo you can scope these to cmd ref only 🙂

We can make issues to update Get Started and Guides later if needed; It's OK if that takes longer. But if you think the work so far on those should be reviewed lmk (we may just need to extract that to 1+ separate PRs).

**Comment by jorgeorpinel** [comment]
And another scope-related Q: is it too much to cover #7089 as well as #3202 and #3211 in a single PR?

**Comment by daavoo** [comment]
> We can make issues to update Get Started and Guides later if needed; It may involve discussing so it's OK if that takes longer and falls on docs team. But if you think the work so far on those should be reviewed, lmk (we may just need to extract that to 1+ separate PRs).

I' ve already included updates for Get Started and 1 of the tables in the user guide. I'll open an issue for the user guide updates.

> And another scope-related Q: is it too much to cover iterative/dvc#7089 as well as #3202 and #3211 in a single PR?

Extracted https://github.com/iterative/dvc.org/pull/3237

File: content/docs/command-reference/exp/show.md
Comment (dberenbaum) [inline content/docs/command-reference/exp/show.md:53]: In general, should we order in the same way they appear (metrics, params, deps) throughout the docs?

Comment (daavoo) [reply content/docs/command-reference/exp/show.md]: I changed the order. 

I have also noticed that `params` and `parameters` are used arbitrarily. 

Should we consolidate `params` / `deps` or the longer version? cc @jorgeorpinel

Comment (iesahin) [reply content/docs/command-reference/exp/show.md]: I usually prefer the longer versions thinking they are more likely to be searched.

**Review by iesahin — APPROVED** [review]
Looks good to me but I'll wait for @jorgeorpinel 's review before merge.

**Comment by jorgeorpinel** [comment]
> I'll wait for @jorgeorpinel 's review before merge

You and @dberenbaum take this one please. I got a few other ones. My initial concerns were addressed. Thanks

File: content/docs/command-reference/exp/show.md
Comment (daavoo) [reply content/docs/command-reference/exp/show.md]: Replaced all occurrences with long version

File: content/docs/start/experiments.md
Comment (dberenbaum) [inline content/docs/start/experiments.md:9]:
```suggestion
metrics, parameters, and dependencies, navigate among them, and commit only the
```

File: content/docs/start/experiments.md
Comment (dberenbaum) [inline content/docs/start/experiments.md:204]:
```suggestion
specific columns using the
```
I think it's clear from the language and the example that you can drop the `Created` column.

File: content/docs/user-guide/experiment-management/comparing-experiments.md
Comment (dberenbaum) [inline content/docs/user-guide/experiment-management/comparing-experiments.md:180]:
```suggestion
the metrics or parameters columns by the option `--sort-by` and
```

File: content/docs/user-guide/experiment-management/comparing-experiments.md
Comment (dberenbaum) [inline content/docs/user-guide/experiment-management/comparing-experiments.md:485]:
```suggestion
have dictionaries as values. `metrics` and `params` dictionaries have keys for
```

File: content/docs/user-guide/experiment-management/comparing-experiments.md
Comment (dberenbaum) [inline content/docs/user-guide/experiment-management/comparing-experiments.md:486]:
```suggestion
each of the metrics or parameters files, and for each file metrics and parameters
```

**Pull request merged**
